lib/song.rb

```diff
@@ -0,0 +1,40 @@
+class Song < ActiveRecord::Base
+  before_create :suggested_date_to_now
```

Comment:
This is an elegant solution, but only if you're _sure_ that you'll never need a song with a different suggested date. Is that the case?